Controllers/CampaignsController.cs

```diff
@@ -92,13 +92,12 @@ public async Task<IActionResult> AddLootByTauheadURL(int campaignId, string url)
             if (model == null) return Conflict();
             return PartialView("_LootCardPartial", model);
         }
-
-        public async Task<IActionResult> Loot(int[] display = null)
+// Added tierSelect and typeSelect
+        public async Task<IActionResult> Loot(int[] display = null, int tierSelect = 0, int typeSelect = 0)

```

Comment:
I'd set defaults to -1. If you set type to 0, you will actually show armors by default, since in C# integer enums are 0-based (see TauManager.Models.Item.ItemType enum).

Reply:
I can change it, but with two different different options related to a single enum (weapons) I couldn't switfly make it enum-based anyway.